src/accumulator/util.rs

```diff
@@ -344,19 +405,23 @@ mod tests {
     }
     #[test]
     fn test_get_proof_positions() {
-        let targets: Vec<u64> = vec![4, 5, 7, 8];
-        let num_leaves = 8;
-        let targets =
-            super::get_proof_positions(&targets, num_leaves, super::tree_rows(num_leaves));
+        use crate::accumulator::util::tree_rows;
```

Comment:
There's no need fro changing this test in this commit right?

Reply:
I think this is a residual from changing `get_proof_positions`, I used to return a tuple, now it's just one vector. I think I changed the naming of the variables, and I also changed the test to something we discussed on #5. But this can be reverted to what we left in #5 without problem.

Reply:
I'd prefer changing this back to #5 before merging.